src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$watch-size: 600px;
+$watch-center-size: $watch-size / 12;
+$watch-center-color: #462239;
+$hour-hand-color: #000;
+$min-hand-color: #141212;
+$sec-hand-color: #910e0e;
+$watch-background-color: radial-gradient(circle,#8f587b 20%, #e8f6c3 100%);
+$box-shadow-hands: 6px 5px 5px 0 rgba(64,41,41,0.75);
```

Comment:
It is not a good idea to create variables for such complex values if you use them just once because it requires the reader to scroll a lot to understand the final result